### DISCUSSION TITLE
mi cuenta ha sido señalada como spam

### DISCUSSION THREAD

**Discussion by estiven2111** [community/community#108623] [Other Feature Feedback, Questions, & Ideas]
### ayuda cuenta marcada como spam

cuenta marcada como spam

### 

al parecer fui hackeado en varias cuentas y la de github me la marcaron como spam he generado un ticket hace mas de 5 dias y no me han respondido les he vuelto a escribir y no me deja github por que dice que tengo un ticket abierto alguien sabe donde hay una comunicacion mas efectiva con el equipo de github o sabe alguien como quitar esta marca de spam muchas gracias

**Comment by ywfan** [comment]
## 回答

这个问题非常典型，涉及到 Agent 交付流程的标准化。补充几点实践经验：

### 1. 交付方式确认清单

在生成任何报告前，建议先确认以下要素：
- 格式偏好: Markdown / PDF / HTML / 飞书文档 / 语雀
- 接收渠道: 本地文件 / 即时消息 / 邮件 / 云盘
- 权限需求: 是否需要公开分享 / 仅团队可见 / 私有
- 时效要求: 实时交付 / 定时汇总 / 紧急优先

### 2. 关于飞书文档 API 的追加模式

这与 Discussion #40 直接相关。飞书文档 API 是 append-only，创建新文档时建议：
1. 先用 create 写入初始内容（标题+摘要）
2. 后续用 append 分段追加，每段 < 5KB
3. 避免用 write 覆盖整个文档（权限不足会 400）

### 3. 语雀知识库管理建议

- 每个知识库控制在 15 个文件以内
- 超过时按主题拆分为独立知识库
- 命名规范：日期-主题-版本，如 2026-06-07-Qwen3-调研-v2

### 4. 通用原则强化

> 报告交付标准流程：生成 → 告知路径 → 询问需求。不要假设交付方式。

这个原则应该写入每个 Agent 的 SOUL.md 或 HEARTBEAT.md 作为强制检查项。
